docs/stencil-docs/installing-stencil-cli/live-previewing-a-theme.md

```diff
@@ -114,9 +98,6 @@ This will install the `npm` modules required to properly leverage the Stencil ev
 
 <!-- theme: info -->
 
-### Themes Prior to 1.10.0

```

Comment:
Can we also get rid of this page https://developer.bigcommerce.com/legacy/stencil-themes/installing-legacy-theme-modules

Reply:
Hi Juned, My understanding is that a lot of the Stencil documentation is old and really needs to be looked at and reviewed.  I would like to create a separate ticket to identify these items.  I would like to close this ticket and open up a new one listing all obsolete information that needs to be removed. I can add this page to it. Would that be okay?

Reply:
Yes this is not a blocker for this one.